package.json

```diff
@@ -40,8 +40,10 @@
     "typescript": "^5.2.2"
   },
   "peerDependencies": {
-    "expo": "*",
     "react": "*",
-    "react-native": "*"
-  }
+    "react-native": "*",
+    "expo": "*",
+    "expo-modules-core": "*"
```

Comment:
Unsure if expo-modules-core needs to be listed as a peer dependency, gotta checkout expo docs!